schemas/schema-schema.ipldsch.json

```diff
@@ -39,31 +39,35 @@
 		},
 		"TypeKind": {
 			"kind": "enum",
-			"members": {
-				"bool": null,
-				"string": null,
-				"bytes": null,
-				"int": null,
-				"float": null,
-				"map": null,
-				"list": null,
-				"link": null,
-				"union": null,
-				"struct": null,
-				"enum": null
+			"representation": {
```

Comment:
Hrm.  I'm a little disquieted by having the whole of the semantic content end up in the representation block, and none in directly on the type.

But I can't think of any alternatives that don't make for fairly staggering redundancy.  So... hrm.  (This might be just a lamenting note rather than a request for anything different.)

Reply:
I've felt the same way about union types.
My first attempt here was to change `members` to `{String:EnumValue}` and have `EnumValue` be a kinded union of `Null null` and `IntValue int` .. but then remembered that we have the same awkward situation with unions and we can more carefully enforce the types if we branch by representation and keep the kinded union out of the mappings.